lib/services/apparmor.pm

```diff
@@ -39,7 +39,13 @@ sub start_service {
 sub check_service {
     # SLE12-SP2 doesn't support systemctl to enable apparmor
     if (is_sle('12-SP3+', get_var('HDDVERSION'))) {
-        systemctl 'is-enabled apparmor.service';
+        # Do double check to avoid perfomance issue
+        my $ret = script_run 'systemctl --no-pager is-enabled apparmor.service';
+        if ($ret) {
+            # If failed try sync and then check again
+            assert_script_run 'sync';
```

Comment:
Lili,
As we discussed, "sleep 2" should be better than "sync" here.

Reply:
I used sleep 2 before, but you said sync is better, I think so too.

Reply:
all right! then no issue from my side